tracing-subscriber/src/fmt/fmt_subscriber.rs

```diff
@@ -604,6 +604,21 @@ impl<C, T, W> Subscriber<C, format::JsonFields, format::Format<format::Json, T>,
             ..self
         }
     }
+
+    /// Sets whether or not the formatter will include newline characters after
+    /// formatting a log event.
```

Comment:
let's also indicate what this does by default. maybe something like this:
```suggestion
    /// Sets whether or not the formatter will include newline characters after
    /// formatting a log event.
    ///
    /// Defaults to `true`, which enables newlines.
```